### ISSUE TITLE
In dark theme "Back" and "hamburger" button icon color contrast with background color is less than 3:1: A11y_.NET maui_User can get all the insights of Dashboard_Non text Contrast

### ISSUE THREAD

**Issue by kapilvaishna** [dotnet/maui#30807]
**Please do not close this bug. This bug should only be closed by Trusted Tester after verification.**
"[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier."

## GitHub Tags
#A11yTCS; #A11yMAS; #WCAG1.4.11; #Non-text Contrast; #BM_.NETMaui_Win32_JUL2025; #DesktopApp; #Win32; #FTP; #A11ySev2; #.NET MAUI SDKs; #A11yWCAG2.1;

## Environment Details:
Application: Developer Balance (.NET Maui)
Windows 11
OS Build: 26100.2161
Version 24H2

## Repro Steps:

1. Install and open the Developer Balance app
2. TAB till any project  and press ENTER key.
3. Apply dark theme
4. TAB till "Back" and "hamburger" button 
5. Observe the issue 

## Actual Result:
In dark theme "Back" and "hamburger" button icon color contrast with background color is less than 3:1.
For back button it is 1.869:1 and for hamburger button it is 2.133:1

**Note:** Similar issue repro with Clear task button

## Expected Result:
"Back" and "hamburger" button icon color contrast with back ground color should be equal and grater than 3:1.

## User Impact:
Low vision user who uses application in dark theme will unable to see "Back" and "hamburger" button icon

### Back button

<img width="1278" height="668" alt="Image" src="https://github.com/user-attachments/assets/2b3c66a9-664f-42d1-8af5-50de8e7dda86" />

### Hamburger button

<img width="1279" height="673" alt="Image" src="https://github.com/user-attachments/assets/0ecd42d9-032b-411b-8f61-daf70c39c3e4" />

### Similar issue

<img width="1279" height="717" alt="Image" src="https://github.com/user-attachments/assets/d4dfdde0-6471-4331-9750-ab2a04763cfb" />

**Comment by ghost** [comment]
This issue has been verified in Visual Studio 17.14.0 Preview 4.0 with MAUI versions (9.0.0 & 9.0.90). It can be reproduced on Windows platform.

**Comment by Vignesh-SF3580** [comment]
@kapilvaishna The issue here is that when switching the UserApp theme, the title bar does not update to reflect the selected theme. This inconsistency causes the title bar to appear mismatched with the rest of the application. The behavior has already been reported in a MAUI issue:
MAUI Issue: https://github.com/dotnet/maui/issues/30518
 
When the system theme is set to dark, the title bar aligns correctly with the UserApp theme and meets accessibility standards. However, when the system theme and UserApp theme differ, the title bar does not adapt, resulting in accessibility issues.
 
Attached are screenshots demonstrating the issue in the following scenarios:

System theme: Light, UserApp theme: Dark
<img width="1417" height="586" alt="Image" src="https://github.com/user-attachments/assets/14d58256-4389-487a-a84a-b8be9b2bd5e2" />

 
System theme: Dark, UserApp theme: Dark
<img width="1417" height="586" alt="Image" src="https://github.com/user-attachments/assets/3004805b-2b18-4109-a2dc-f2b857e5eb5f" />

**Comment by kapilvaishna** [comment]
@Vignesh-SF3580 Thank you for explaining the issue. Once the MAUI issue (https://github.com/dotnet/maui/issues/30518) is resolved, we will verify it and then proceed to close this bug.

**Comment by jsuarezruiz** [comment]
> [@Vignesh-SF3580](https://github.com/Vignesh-SF3580) Thank you for explaining the issue. Once the MAUI issue ([#30518](https://github.com/dotnet/maui/issues/30518)) is resolved, we will verify it and then proceed to close this bug.

After merge the fix https://github.com/dotnet/maui/pull/30525, could we process to verify it with the Developer Balance App?

**Comment by kapilvaishna** [comment]
Issue is repro in below version 

<img width="256" height="97" alt="Image" src="https://github.com/user-attachments/assets/df08ccb5-6e25-4899-9d01-9df0d8d3b48f" />


<img width="1280" height="768" alt="Image" src="https://github.com/user-attachments/assets/9d915e87-51e7-46b3-bf39-1d9efbf265e4" />

<img width="1280" height="768" alt="Image" src="https://github.com/user-attachments/assets/22b96b87-5d35-4f69-9d94-3aa5b4430beb" />

**Comment by AnKushSingh05** [comment]
#Closed;#A11yFixed;
Issue is fixed

<img width="1366" height="768" alt="Image" src="https://github.com/user-attachments/assets/726b8151-0091-46f2-9e2e-b9270aa039b8" />

<img width="1366" height="768" alt="Image" src="https://github.com/user-attachments/assets/24562608-9304-4757-bfce-5a00b6c471d2" />
